packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.7.6
+
+* Changed storage location for captured images and videos to internal cache on Android,
+to comply with new Google Play storage requirements.
```

Comment:
I am wondering if this should be considered as a breaking change. As the API itself doesn't change, but the locations where images from the camera are stored is changed to scoped storage instead of the [Environment.DIRECTORY_PICTURES](https://developer.android.com/reference/android/os/Environment#DIRECTORY_PICTURES) external file location. 
Meaning it will be more difficult for other Apps to reach the images. @stuartmorgan or @cyanglaz can you possibly advice on this?

Reply:
It's probably better to err on the side of making it a breaking change since we don't know if people are relying on the location, even if it's not something that's guaranteed by the API.

@blasten any preference on this?

Reply:
> "If you require your picked image to be stored permanently, it is your responsibility to move it to a more permanent location."  

I agree the plugin never said that this was the case in the first place, but I'm inclined to think that given the nature of the issue, making it a major release sounds good.

Reply:
The changelog and pubspec have been updated to show this as a breaking change.